README.md

```diff
@@ -214,7 +221,14 @@ configured explicitly when invoking Docker.
 Applications that run under Docker are usually easy to run on
 OpenShift.  There will be configuration changes.  OpenShift
 configuration is beyond the scope of this readme.  The *runDocker.sh*
-script is a model for what OpenShift needs to be configured to supply.
+script is a model for what the OpenShift environment needs to supply for correct configuration.
+
+The version of kartograafr to run is specified in the OpenShift build configuration.  
+It should point to a git tag for testing and production instances.
+
+A new build can be started with the "new build" button on the build configuration page.
```

Comment:
Can the operations done in the OpenShift console webapp also be done via its CLI?